File: course_js210_computational_thinking_and_javascript_programming/lesson_1_javascript_basics/book_introduction_to_javascript_programming/input_output/exercises/age.js
var rlSync = require('readline-sync');

let age = rlSync.question("What's your age?");
age = parseInt(age)
console.log(`You are ${age} old`);
console.log(`In 10 years, you will be ${age + 10} years old`);
console.log(`In 20 years, you will be ${age + 20} years old`);
console.log(`In 30 years, you will be ${age + 30} years old`);
console.log(`In 40 years, you will be ${age + 40} years old`);

File: course_100/introduction_to_programming_with_ruby_ls_book/Chpt 2 - Variables/age.rb
# Write a program called age.rb that asks a user how old they are and then tells them how old they will be in 10, 20, 30 and 40 years. Below is the output for someone 20 years old.
require "pry"

puts "How old are you"
age = gets.chomp
age = age.to_i

# binding.pry
puts "In 10 years you will be " + (age + 10).to_s + "!"
puts "In 20 years you will be " + (age + 20).to_s + "!"
puts "In 30 years you will be " + (age + 30).to_s + "!"
puts "In 40 years you will be " + (age + 40).to_s + "!"
